### PR TITLE
data: anthonrodgrs01

### PR DESCRIPTION
**Pull request by anthonrodgrs01**
## Check List (Check all the applicable boxes) 

- [yes ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ yes] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4501"></a>

### CODE REVIEW

**Review by AbbeyIT — APPROVED** [review]
LGTM! Thank you for your contribution.

File: data/anthonrodgrs01.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Anthon Rodrigues",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Software Developer, Problem solver, Getting started with Open source",
+  "avatar": "https://github.com/anthonrodgrs01.png",
```
Comment (kumarsonsoff3) [inline data/anthonrodgrs01.json:6]: In our recent update, we no longer need the `avatar` key. So please remove this from your json file. 
```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by himanshu1221 — APPROVED** [review]
LGTM:+1:

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for your Contribution! 🎉 

<p align="center"></p>

**Review by Hrishi5111998 — APPROVED** [review]

**Comment by anthonrodgrs01** [comment]
I am really new to open source even after all the checks it was showing the option for updating the branch so I clicked it, I am sorry if the maintainer had to go over again for me... I am really new to this.. I dont know if my pull request is merged or not

**Comment by anthonrodgrs01** [comment]
Can anyone merge my pull request

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**